src/services/findAddress.js

```diff
@@ -0,0 +1,15 @@
+import createNewHouse from '../models/house'
+
+export const findAddress = async (addressData) => {
+    const {country, city, street, houseNr, flatNr} = {...addressData}
+    const address = await createNewHouse.findOne({
+      'address.country': country,
+      'address.city': city,
+      'address.street': street,
+      'address.houseNr': houseNr,
+      'address.flatNr': flatNr
+    })
+    if (address === null) return { status: 'success'};
```

Comment:
Should it return anything? In the controller the result of this function isn't assigned to any variable.